### DISCUSSION TITLE
Observable markdown cell: is it possible to use it?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#870] [Q&A] [answered]
Hi,
in observable there are also markdown cells. How to use them in quarto.

It would be great to use them to write markdown text coupled with variables from other cells (I have written about this [here](https://github.com/quarto-dev/quarto-cli/discussions/865)).

Something like:

~~~
```{ojs}
viewof date = Inputs.date({label: "Date", value: "2022-05-13"})
```

```{ojs markdown}
# Title

Today is ${date}
```
~~~

Thank you

**Comment by jjallaire** [comment, marked as answer]
You can use `${date}` anywhere in markdown. For example, the following should work:

````markdown
```{ojs}
viewof date = Inputs.date({label: "Date", value: "2022-05-13"})
```

Today is ${date}
````

**Reply by aborruso** [reply]
I'm stupid, thank you :)
